org.scala-ide.sdt.core/src/org/scalaide/ui/internal/jdt/model/ScalaOverrideIndicatorBuilder.scala

```diff
@@ -35,28 +39,68 @@ case class JavaIndicator(scu: ScalaCompilationUnit,
   }
 }
 
-trait ScalaOverrideIndicatorBuilder { self : ScalaPresentationCompiler =>
```

Comment:
just a question, what was the reason to make `ScalaIndicator` a part of this trait? I see that presentation compiler can be retrieved from `ScalaCompilationUnit`.